android/app/src/main/java/org/fossasia/openevent/data/Microlocation.java

```diff
@@ -24,69 +30,10 @@
     private int floor;
     private String room;
 
-    public Microlocation() {
-    }
+    public Microlocation() {}
```

Comment:
I have already tried it but it gives error.
Error:Class "Microlocation" must declare a public constructor with no arguments if it contains custom constructors.

Microlocation class has one constructor with two arguments (int id, String name) because of it it is giving error.